docs/docs/ecommerce-tutorial/index.md

```diff
@@ -194,7 +195,7 @@ const Checkout = class extends React.Component {
             console.log("Transaction processed successfully")
             this.resetButton()
             this.setState({ paymentMessage: "Payment Successful!" })
-            return res.json()
+            return res
```

Comment:
Is this needed too?

Reply:
I did need to do that in order to get it to work. I was getting a parsing error when I left it in.